.travis.yml

```
@@ -55,23 +56,31 @@ script:
 
 jobs:
   include:
-    - stage: Validations
+    - stage: linting
       before_script:
         - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
       script:
         - ./scripts/verify.sh
+    - stage: golden
```

```suggestion
    - stage: generate
```

IHMO:  Golden has no meaning and we have the target that `make generate` which gen the testdata and do all required tricks as, for example, ensure the correct module versions which should be used. (Described  in the CONTRIBUTION.md)